cgp/genome.py

```diff
@@ -359,7 +366,11 @@ def change_address_gene_of_output_node(self, new_address: int, output_node_idx:
         self.dna = dna
 
     def set_expression_for_output(
-        self, dna_insert: List[int], hidden_start_node: int = 0, output_node_idx: int = 0
+        self,
+        dna_insert: List[int],
+        hidden_start_node: int = 0,
+        output_node_idx: int = 0,
+        target_expression: Optional[str] = None,
```

Comment:
i would force users to always provide a target expression for their own benefit by making this *not* optional. we all need help not shooting ourselves in the foot sometimes ;)